src/adapters/telegram/helpers/client.ts

```diff
@@ -28,10 +29,10 @@ export const telegramFormattedNotifier = (messagePayload: TLMessageFormattedPayl
 
     const sendInterval = setInterval(async () => {
       clearInterval(sendInterval);
-      await bot.telegram.sendMessage(chatIds[currentElem], text, { parse_mode: parseMode });
+      await telegram.telegram.sendMessage(chatIds[currentElem], text, { parse_mode: parseMode });
```

Comment:
Seems unusual that you have this notation `telegram.telegram` this should be fixed.

Reply:
it would be fixed in https://github.com/ubiquity/bounty-bot/pull/85